src/types.ts

```diff
@@ -53,6 +53,7 @@ export interface ResultCardOptions {
     },
     index: number
   ) => JSX.Element;
+  getResultCardImageUrl?: (imageUrl: string) => string;
```

Comment:
Should we pass the item data as well? 

In case some customers have logic that depends on some metadata.

Reply:
Great callout